view/frontend/layout/checkout_index_index.xml

```diff
@@ -104,6 +104,9 @@
                                                                         <item name="omise_offsite_shopeepay" xsi:type="array">
                                                                             <item name="isBillingAddressRequired" xsi:type="boolean">true</item>
                                                                         </item>
+                                                                        <item name="omise_offsite_paypay" xsi:type="array">
+                                                                            <item name="isBillingAddressRequired" xsi:type="boolean">true</item>
```

Comment:
How does this interact with the fact that on our API PayPay has billing address as optional?

Reply:
I believe this is frontend UI config to show billing address component on checkout page.

<img width="300" alt="Screenshot 2023-05-02 at 4 57 55 PM" src="https://user-images.githubusercontent.com/108650842/235636945-0f9fea12-d86b-4311-a28f-5f007dd17cee.png">